nightfall-client/client.env

```diff
@@ -0,0 +1,4 @@
+ETH_NETWORK=goerli
```

Comment:
Call this file something else `client-example.env`. Add ´client.env` to .gitignore. In the instructions, then specify that one should copy `client-exmaple.env` to `client.env` and add their own information